### DISCUSSION TITLE
What VSCode built in feature gave the "Commit Changes" in right click dropdown?

### DISCUSSION THREAD

**Discussion by agreedSkiing** [VSCodium/vscodium#1904] [Q&A]
See title and as reference a picture of the "Commit changes" contents
![bild](https://github.com/VSCodium/vscodium/assets/93997033/35be5542-5d40-4859-8466-85516ffae7ec)

**Comment by agreedSkiing** [comment]
Was missing the GitLens extension

**Reply by daiyam** [reply]
Ya, it has happened for me too...
